### PR TITLE
Fix version, because 6.0.0-beta1 does not exist anymore

### PR DESCRIPTION
**Pull request by dhoppe**
#### Pull Request (PR) description
This is necessary to fix the Travis CI pipeline.

#### This Pull Request (PR) fixes the following issues
Fixes #160 
Fixes #161 
Fixes #162 
Fixes #164

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -104,8 +104,8 @@
               'beta'   => 'https://packages.grafana.com/oss/rpm-beta',
             }
 
-            yumrepo { 'grafana':
-              descr    => 'grafana repo',
```
Comment (alexjfisher) [inline manifests/install.pp]: Do we need to remove the old repo?

```puppet
yumrepo { 'grafana':
  ensure => absent,
}
```
??

Comment (dhoppe) [reply manifests/install.pp]: You are probably right. I will fix this in the evening.

File: manifests/install.pp [hunk 2]
```diff
@@ -105,7 +105,13 @@
             }
 
             yumrepo { 'grafana':
-              descr    => 'grafana repo',
+              ensure => 'absent',
+              descr  => 'grafana repo',
+              before => Package[$::grafana::package_name],
```
Comment (bastelfreak) [inline manifests/install.pp]: please do not prefix variables with `::` after `$`. that is legacy puppet3 syntax

File: manifests/install.pp [hunk 3]
```diff
@@ -105,7 +105,13 @@
             }
 
             yumrepo { 'grafana':
-              descr    => 'grafana repo',
+              ensure => 'absent',
+              descr  => 'grafana repo',
```
Comment (alexjfisher) [inline manifests/install.pp]: Nit. `descr` not needed.

File: manifests/install.pp (hunk 2 above)
Comment (dhoppe) [reply manifests/install.pp]: This will be fixed, after #164 has been merged, because it will introduce the necessary lint check.

**Pull request merged**